src/backend/main.py

```diff
@@ -1,25 +1,68 @@
 import glob
+import sys
+import os
+import argparse
+from pipeline.templates import full_pipeline, default_pipeline
+from pipeline.mat import Mat
+from google.cloud import storage
 from pipeline.templates import full_pipeline, default_pipeline, training_pipeline, nutrient_pipeline
 import firebase_admin
 from firebase_admin import credentials, firestore
 import asyncio
 from pipeline.mat import Mat, Channels
+from pipeline.config import CloudConfig
 import numpy as np
 
 def main():
     """Main entry point."""
 
+    # parse arguments
+    parser = argparse.ArgumentParser(description='Process cloud arguments')
+    parser.add_argument('--path', dest='path', default=None)
+    parser.add_argument('--mode', dest='mode', default=None)
+    namespace = parser.parse_args(sys.argv[1:])
+    path = namespace.path
+    mode = namespace.mode
+
+    # check if we use cloud data or local data
+    if path is None or mode is None:
+        imgs = [Mat.read(file) for file in glob.glob("pipeline/test/data/mosaicing/farm/D*.JPG")]
+        path = "pipeline/test/data/mosaicing/farm/D*.JPG"
+        cloud_config = CloudConfig()
```

Comment:
Doesn't this escape the scope? Since you initialize this in both the `if` and `else` perhaps put this in front

> cloud_config: CloudConfig = CloudConfig()